website/content/en/docs/best-practices/managing-resources.md

```diff
@@ -0,0 +1,191 @@
+contention on a node, pods will start getting evicted and possibly killed according to their OOM score. The node will be flagged with a
+MemoryPressure condition and eventually made unschedulable. However, when there is CPU contention on the node, neighbour pods may
+get slowed down to the CPU cores they requested.
+
+However, a popular practice by cluster administrators is to leverage [ResourceQuota][k8s-resources-quotas] to limit
+the total amount of resources that can be requested or allowed in a single namespace.
+This may protect against over consumption of resources by operands of a faulty or wrongly configured operator.
+On the other hand it also means that the operator may not be able to create additional pods,
+limiting its functionality when the limit has been reached.
+
+Also, see might want to check in the K8s docs the following sections:
+- [Motivation for CPU requests and limits][k8s-motivations]
+- [Motivation for Memory requests and limits][k8s-motivations-memory]
+- [If you do not specify a memory limit][k8s-missing-memory-limits] 
+- [If you do not specify a CPU limit][k8s-missing-cpu-limits]
+
+### What happens when:
+
+#### Limits reached
+
+_What happens when the resource limits have been reached?_
+
+- `For Memory`: the container might be terminated with the reason `OOM Killed`. If it is restartable, the kubelet will
+  restart it, as with any other type of runtime failure.
+- `For CPU`: the container might or might not be allowed to exceed its CPU limit for extended periods of time.
+  However, it will not be killed for excessive CPU usage. The `CPU` is considered a "compressible" resource,
+  and if the `Pod` starts hitting the CPU limits, Kubernetes uses kernel to starts throttling the container.
+  That means the CPU will be artificially restricted, giving a potentially worse performance only.
+
+You might want to check the [Troubleshooting][k8s-troubleshooting] section in the Kubernetes documents to better
+understand how to debug these scenarios.
+
+#### Limits are specified but not requests
+
+_What happens when the resource limits are defined but not the requests?_
+
+If you specify a CPU or Memory limit for a Container but do not specify a request, 
+Kubernetes automatically assigns a CPU or Memory request that matches the limit. 
+In this way, you will be requesting **always** the limit and will be allocating 
+more resources than required. (**NOT RECOMMENDED**)
+
+#### Values are too big
+
+Memory and CPU requests and limits are associated with Containers, but be aware that the Memory and CPU 
+requests and limits of a Pod are the sum of its specific computing types for all the containers in the Pod. 
+
+If you define that your Pods should have Memory or CPU request too big then, you might not only be allocating and 
+blocking the usage of more than you ought unnecessarily. Also, your Operator consumers might be unable to install them 
```

Comment:
saying consumers won't be able to install them is a bit confusing.

yes technically the OLM install may fail, because the operator deployment will never be able to get scheduled (if no node has sufficient resources to meet the request), but i'd say it's more that consumers won't be able to "deploy" the operator than that they won't be able to "install" it.  The operator installs, the deployment of the operator fails.

Reply:
Tried to address it. Please, feel free to re-check.